index.js

```diff
@@ -149,10 +155,15 @@ SQLdown.prototype._get = function (key, options, cb) {
     if (!res.length) {
       return cb(new Error('NotFound'));
     }
+
     try {
       var value = JSON.parse(res[0].value);
       if (asBuffer) {
-        value = new Buffer(value);
+        if(value.type === 'Buffer'){
```

Comment:
that sounds good, but to be clear right now we're not storing the buffers as utf or base64, we're storing them as `JSON.stringify`. https://github.com/calvinmetcalf/SQLdown/issues/19#issuecomment-90731995